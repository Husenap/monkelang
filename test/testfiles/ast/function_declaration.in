monke work concat want string first and string second give string monke do
  monke has number x is 42
monke done

