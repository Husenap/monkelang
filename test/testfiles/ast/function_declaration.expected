FuncDeclStmt: name:concat return:string
  Params: 
    IdDeclStmt: id:first type:string
    IdDeclStmt: id:second type:string
  Body: 
    BlockStmt: 
      IdDeclStmt: id:x type:number
        NumberExpr: 42
